zgw/notificaties-api/src/main/kotlin/com/ritense/notificatiesapi/NotificatiesApiAutoConfiguration.kt

```diff
@@ -16,9 +16,17 @@
 
 package com.ritense.notificatiesapi
 
+import com.ritense.plugin.service.PluginService
+import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 
 @Configuration
 class NotificatiesApiAutoConfiguration {
 
+    @Bean
+    fun notificatiesApiPluginFactory(
```

Comment:
Missing: @ConditionalOnMissingBean(...)
So that implementation can decide to override this bean

Reply:
addressed. mind you this annotation is also missing in catalogi-api plugin.